File: VTV3008.py
from math import sqrt

def solve(i):
    if i < 0:
        return 0
    if ((float)(arr[i])) % 2 == 0: 
        return (float)(arr[i]) + solve(i-1)
    else:
        return solve(i-1)
    
def output(n):
    print(n)

testcase = (int)(input())
a = input().split(' ')
arr = []
for i in range(0,testcase):        
    arr.append((float(a[i])))
n = solve(testcase-1)
output(n)

File: VTV3008.java
import java.util.Scanner;

public class Main {
    
    public static int solve(int i, String[] arr) {
        if (i < 0) {
            return 0;
        }
        int a = Integer.parseInt(arr[i]);
        if (a % 2 == 0) {
            return a + solve(i - 1, arr);
        } else {
            return solve(i - 1, arr);
        }
    }
    
    public static void output(int n) {
        System.out.println(n);
    }
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int testcase = Integer.parseInt(scanner.nextLine());
        String[] arr = scanner.nextLine().trim().split(" ");
        
        int n = solve(testcase - 1, arr);
        output(n);
        
        scanner.close();
    }
}
